testingutils/src/test/java/com/github/awsjavakit/testingutils/hamcrest/InstantIsCloseToTest.java

```diff
@@ -0,0 +1,45 @@
+package com.github.awsjavakit.testingutils.hamcrest;
+
+import static com.github.awsjavakit.testingutils.hamcrest.InstantIsCloseTo.closeTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.time.Duration;
+import java.time.Instant;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+class InstantIsCloseToTest {
+
+  @Test
+  void shouldReturnTrueIfActualIsEqualToExpected() {
+    var expected = Instant.parse("1980-01-01T00:00:00z");
+    var actual = Instant.parse("1980-01-01T00:00:00z");
+    assertThat(actual, is(closeTo(expected, Duration.ofSeconds(1))));
+  }
+
+  @Test
+  void shouldReturnFalseIfActualIsNotEqualToExpected() {
```

Comment:
shouldReturnFalse or shouldThrowException? The name says false, but the test tests for exception

Reply:
fixed